src/services/actions/todo.js

```diff
@@ -4,3 +4,12 @@ export const addTodo = (props) => {
     payload: props,
   };
 };
+
+export const deleteTodo = (id) => {
+  return {
+    type: "DELETE",
+    payload: {
+      id: id,
```

Comment:
@kaa-wtag Why are we taking the payload as an object?

Reply:
There was not any specific reason, changed it, passing the id now. Thanks.